corehq/apps/userreports/dbaccessors.py

```diff
@@ -0,0 +1,10 @@
+def get_report_configs_by_data_source(domain, data_source_id):
```

Comment:
Ah ok, good to know. I didn't realize there were two styles. I'll go with the second in the future.